exercises/twelve-days/twelve_days.exs

```diff
@@ -0,0 +1,61 @@
+defmodule TwelveDays do
+  @doc """
+  Given a `number`, return the word representation of it, or "a" for one.
+
+  1 => "a",
+  2 => "two",
+  3 => "three",
+  etc...
+  """
+  @spec count(number :: integer) :: String.t()
+  def count(number) do
+  end
+
+  @doc """
+  Given a `number`, return the ordinal representation of it.
+
+  1 => "first",
+  2 => "second",
+  3 => "third",
+  etc...
+  """
+  @spec ordinal(number :: integer) :: String.t()
+  def ordinal(number) do
```

Comment:
Same thing here - this feels more like implementation specifics than actual usable behavior.